app/assets/stylesheets/game_show.scss

```diff
@@ -0,0 +1,22 @@
+.tile {
+  z-index: 100;
+  cursor: pointer;
+}
+
+.black {
+  float: left;
+  width: 80px;
+  height: 80px;
+  background-color: #000;
+}
+
+.white {
+  float: left;
+  width: 80px;
+  height: 80px;
+  background-color: #fff;
+}
+
+.selected_piece {
```

Comment:
With CSS, apply the logic above for JS naming vs Rails, except use dashes instead of underscores. `selected-piece`